navigator_engine/common/dash_app.py

```diff
@@ -0,0 +1,14 @@
+import dash
+import flask
+from navigator_engine.common import graph_visualizer
+
+
+def add_dash_app(navigator_app):
+    dash_app = dash.Dash(__name__, server=navigator_app, url_base_pathname='/graph/')
+    graph_visualizer.create_visualizer(flask_app=navigator_app, dash_app=dash_app)
+
+    @navigator_app.route('/graph')
+    def graph():
+        return flask.redirect('/graph')
```

Comment:
I don't understand what is happening here... can you help me out?  It looks like you are putting redirect at this url to itself?  I would expect this to cause an eternal loop of redirects.

Where/how in this little endpoint does it hook into the dash app?

Reply:
The endpoint is configured in this line in dash_app.py: `dash_app = dash.Dash(__name__, server=navigator_app, url_base_pathname='/graph/')`

I removed the redirect you mentioned and it had no effect on the functionality. Seems like I just included without fully understanding the docs I was reading.